src/dappsExplorer/TabDiscover.tsx

```diff
@@ -237,7 +251,13 @@ function TabDiscover({ navigation }: Props) {
                   </Text>
                 }
                 <DappFeaturedActions onPressShowDappRankings={handleShowDappRankings} />
-                {getFeatureGate(StatsigFeatureGates.SHOW_STABLECOIN_EARN) && <EarnCta />}
+                {getFeatureGate(StatsigFeatureGates.SHOW_STABLECOIN_EARN) ? (
```

Comment:
nit: i think it's clearer to move the business logic of what's enabled above (outside the ts embedded in the tsx).  it took me a sec to parse the nested ternary operators.  you could do:

ts:
```ts
let earnInstance = null
const showStablecoinEarn = getFeatureGate(StatsigFeatureGates.SHOW_STABLECOIN_EARN)
if (showStablecoinEarn) {
  earnInstance = ...
}
```
tsx:
```tsx
{earnInstance}
```

Reply:
Moved the conditional rendering to a hook in 532eac80e94c101bd66770e017f5dccd8f412a26. Hopefully this makes it a bit clearer what the business logic is for displaying the earn component.